product-pricing-service/src/main/java/com/hedvig/productPricing/campaign/integration/contract/model/MarketCurrency.kt

```diff
@@ -2,5 +2,6 @@ package com.hedvig.productPricing.campaign.integration.contract.model
 
 enum class MarketCurrency {
```

Comment:
Didn't know about this enum, maybe we should add the currency unit here? 🤔 
```suggestion
enum class MarketCurrency (val currencyUnit: CurrencyUnit) {
```

Reply:
Should we add even do we don't currently have a plan to use it?